code/edgeconsent/build.gradle

```diff
@@ -222,12 +189,12 @@ task platformUnitTestJacocoReport(type: JacocoReport, dependsOn: "testPhoneDebug
     additionalClassDirs.setFrom files([debugTree])
```

Comment:
If there are no ADB prefixed classes for this repo source code, we can remove that from excludeRegex (same for platformFunctionalTestJacocoReport)

Reply:
We have ADBCountDownLatch class.

Reply:
The ADBCountDownLatch class is a utility class used for testing and is not included in the coverage report. There should not be any "ADB" classes in the main (non-test) source set. Can you remove the "ADB" exclusion from the coverage configuration?